src/containers/HeaderContainer/HeaderContainer.vue

```diff
@@ -249,13 +268,15 @@ export default {
       gasPrice: '0',
       error: {},
       resolver: () => {},
-      showGettingStarted: ''
+      showGettingStarted: '',
+      address: '',
```

Comment:
Not needed as this is set in account global object.

Reply:
Removed